packages/sui-test/bin/karma/babelPatch.js

```diff
@@ -0,0 +1,44 @@
+module.exports = function (babel) {
```

Comment:
maybe this file might be in sui-codemods renamed as `sui-test-describe-context-patch`?